ep/build-velox/src/get_velox.sh

```diff
@@ -16,8 +16,8 @@
 
 set -exu
 
-VELOX_REPO=https://github.com/oap-project/velox.git
-VELOX_BRANCH=2025_04_14
+VELOX_REPO=https://github.com/JkSelf/velox.git
```

Comment:
Is there related velox pr?

Reply:
@zjuwangg This is the velox pr https://github.com/facebookincubator/velox/pull/13041